### PR TITLE
Meta information for VSNetBeans 13.0.301

### PR DESCRIPTION
**Pull request by MartinBalin**
Preparing release of VSCode NetBeans extension 13.0.301 from branch [vsnetbeans_preview_1303](https://github.com/apache/netbeans/tree/vsnetbeans_preview_1303)

### CODE REVIEW

File: meta/netbeansrelease.json [hunk 1]
```diff
@@ -666,7 +666,7 @@
     },"release1303": {
         "position": "18",
         "ant": "ant_latest",
-        "jdk": "jdk_11_latest",
+        "jdk": "jdk_1.8_latest",
```
Comment (jtulach) [inline meta/netbeansrelease.json:669]: I think this is more accurate, especially when [netbeans-vscode](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/) job still uses JDK8.

**Review by JaroslavTulach — APPROVED** [review]
Looks fine. The [build run](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/1029/console) properly discovered `metabuild.RawVersion=13.0.301`, so everything seems to work the same way it used to when releasing `12.6.301`.

**Comment by ebarboni** [comment]
looks fine, I do some commit to have [netbeans-tlp ](https://ci-builds.apache.org/job/Netbeans/job/netbeans-TLP/job/netbeans/) job able to deal with vscode branches for

File: meta/netbeansrelease.json [hunk 2]
```diff
@@ -663,6 +663,35 @@
             "month": "11",
             "year": "2021"
         }
+    },"release1303": {
```
[the remark this answers is not in the document]
Comment (MartinBalin) [reply meta/netbeansrelease.json]: Fixed.

**Review by ebarboni — APPROVED** [review]

**Comment by ebarboni** [comment]
I'm sorry for the lack of comment in the build system but the month release date should contains a 2 digit but should be put **after** release because
If release month is invalid (- or 1 digità it will build the full stack and binaries
If release month is valid the build generate only the apidoc. (allow the historic version browsability)

On merge https://ci-builds.apache.org/job/Netbeans/job/netbeans-TLP/job/netbeans/job/vsnetbeans_preview_1303/ should do the same work as a standard release. 

All the best.

**Pull request merged**